example/src/Utilities/payByID.js

```diff
@@ -0,0 +1,29 @@
+import { AdyenCSE, AdyenAction } from '@adyen/react-native';
+import { ENVIRONMENT } from '../Configuration';
+import ApiClient from './APIClient';
+import { checkoutConfiguration } from './AppContext';
+import { isSuccess } from './Helpers';
+
+export async function payByID(id, cvv, configuration) {
+  const encryptedCard = await AdyenCSE.encryptCard(
+    { cvv: cvv },
```

Comment:
It can be shortened  to `{ cvv },`.

Reply:
Fixed [97323d1](https://github.com/Adyen/adyen-react-native/pull/342/commits/97323d11bbb6a2b364c930e4265297ee7a598917)